addons/base_import_module/models/ir_module.py

```diff
@@ -235,7 +235,7 @@ def import_zipfile(self, module_file, force=False):
                             if self._import_module(mod_name, path, force=force):
                                 success.append(mod_name)
                         except Exception as e:
-                            _logger.exception('Error while importing module')
+                            _logger.warning('Error while importing module')
```

Comment:
warning => bad thing happened but the server was able to recover and proceed the processing
error => bad thing happened and the server wasn't able to proceed the processing
unless sentry matches on errors and not tracebacks

I would still log what module failed and why
```suggestion
                            _logger.error('Error while importing module %r: %s', mod_name, e)
```

Reply:
Hello @Julien00859 

If we use 'logger.error', it will still receive as trace-back in sentry.

cc @yelizariev

Reply:
ok I get it for error=>warning, can you at least log the module name and the exception title?

Reply:
Done :+1:

Reply:
> If we use 'logger.error', it will still receive as trace-back in sentry.

This is not a relevant reason for a change of log type. I agree that sometimes the `error`/ `exception` level is incorrectly used and that we should change it, but not always. When these log types are legit and we still want to ignore it on Sentry, we have the possibility to add some filters in our Sentry code 

Here, considering Julien's feedback, the log type could become `error`, if we really want it, else we could also stop here and simply close the PR (@Julien00859 the floor is yours)

We will ignore that log on our side (cc @abz-odoo )